recipes/gperf/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+
+class GperfConan(ConanFile):
+    name = "gperf"
+    license = "GPL-3.0-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/gperf"
+    description = "GNU gperf is a perfect hash function generator"
+    topics = ("conan", "gperf", "hash-generator", "hash")
+    settings = "os_build", "arch_build", "compiler"
+    _source_subfolder = "source_subfolder"
+    _autotools = None
+
+    @property
+    def _is_msvc(self):
+        return self.settings.compiler == "Visual Studio"
+
+    @property
+    def _is_mingw_windows(self):
+        return self.settings.os_build == "Windows" and tools.os_info.is_windows and self.settings.compiler == "gcc"
+
+    def build_requirements(self):
+        if self.settings.os_build == "Windows" and tools.os_info.is_windows:
+            if "CONAN_BASH_PATH" not in os.environ and tools.os_info.detect_windows_subsystem() != 'msys2':
+                self.build_requires("msys2/20190524")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_autotools(self):
+        if not self._autotools:
+            args = []
+            cwd = os.getcwd()
+            win_bash = self._is_msvc or self._is_mingw_windows
+            self._autotools = AutoToolsBuildEnvironment(self, win_bash=win_bash)
+            if self._is_msvc:
+                args.extend(["CC={}/build-aux/compile cl -nologo".format(cwd),
+                            "CFLAGS=-{}".format(self.settings.compiler.runtime),
+                            "CXX={}/build-aux/compile cl -nologo".format(cwd),
+                            "CXXFLAGS=-{}".format(self.settings.compiler.runtime),
+                            "CPPFLAGS=-D_WIN32_WINNT=_WIN32_WINNT_WIN8 -I/usr/local/msvc32/include",
+                            "LDFLAGS=-L/usr/local/msvc32/lib",
```

Comment:
are these -L and -I really needed? other recipes don't use it. I don't like to use hard-coded paths unless there is a strong reason

Reply:
This comes from the conan-community recipe. 
I can remove them and my local build still succeeds, so they might not be necessary

Reply:
okay, let's try to remove that. I believe such directories even don't exist in our MSYS distribution.